deel/lip/layers.py

```diff
@@ -949,6 +952,8 @@ def __init__(
             raise RuntimeError("stride must be equal to pool_size")
         if padding != "valid":
             raise RuntimeError("NormalizedConv only support padding='valid'")
+        if eps_grad_sqrt < 0.0:
```

Comment:
Should be `<= 0.0`.

Reply:
`eps=0` works under some circumstances, like the test case with synthetic data